Libraries/plist/Sources/UnixTime.cpp

```diff
@@ -16,8 +16,13 @@ using plist::UnixTime;
 void UnixTime::
 Decode(uint64_t in, struct tm &out)
 {
+#if _WIN32
+    time_t const t = in;
```

Comment:
It appears that `gmtime_s` is part of an [optional][0] "secure programming" library in C11, detectable by the `__STDC_LIB_EXT1__` macro. Microsoft appears to have adopted them quite strongly (see *note*), but they seem to be [widely][1] [disliked][2] elsewhere so unlikely to appear on macOS or Linux.

[0]: http://stackoverflow.com/a/16700951
[1]: https://en.wikipedia.org/wiki/C11_(C_standard_revision)#Criticism
[2]: http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1967.htm

*Note*: By default, MSVC issues a warning for the "non-secure" variants unless `_CRT_SECURE_NO_WARNINGS` is defined; that's added through `CMakeLists.txt` in the build configuration patch.

Also, C++11 is based on C99. Only C++17 will change that to C11, so even if it was required in C11 it wouldn't necessarily be usable here.